### ISSUE TITLE
Update or replace djangorestframework-datatables

### ISSUE THREAD

**Issue by Jawayria** [edx/upgrades#16]
We use the package [djangorestframework-datatables](https://github.com/izimobil/django-rest-framework-datatables) in edx/ecommerce. It hasn't yet added support for Django3.2. Please follow the guidance in https://openedx.atlassian.net/wiki/spaces/AC/pages/3036972032/Handling+Outdated+Dependencies to resolve the problem this poses for the Open edX Django 3.2 upgrade.

**Comment by MoisesGSalas** [comment]
I opened a [PR](https://github.com/izimobil/django-rest-framework-datatables/pull/116) with tests for django 3.2. I'll open an issue for the version bump once it's merged.

**Comment by natabene** [comment]
@MoisesGSalas Thanks so much for looking into this, I appreciate. I will assign this to you. Please update us on your progress here.

**Comment by MoisesGSalas** [comment]
I'm yet to receive an answer from the maintainer, so I deceided to run the tests for Django 3.2 by myself against the latest release on PyPI (the original PR have tests for a pending release).

You can see the run [here](https://github.com/eduNEXT/django-rest-framework-datatables/actions/runs/1244310022). I branched from commit [`979370c`](https://github.com/izimobil/django-rest-framework-datatables/commit/979370ca6e8b2f3431b2ad5feb069e887eae3a29), which is tagged as v0.6.0 (latest on PyPI).

I don't know if this is enough for you, but for the time being at least we know the tests are green for Django 3.2 on the current release.

**Comment by natabene** [comment]
@MoisesGSalas Thanks so much for an update. @Jawayria, @awais786  What do you think?

**Comment by iamsobanjaved** [comment]
Yeah, sure we can count this release `0.6.0` in Django 3.2 supported releases and can update our dependency sheet with a note that this release is tested with Django 3.2 and support in further new releases depends on the merge status of this [PR](https://github.com/izimobil/django-rest-framework-datatables/pull/116)
